tests/callbacks/test_xla_stats_monitor.py

```diff
@@ -30,31 +31,36 @@ def test_xla_stats_monitor(tmpdir):
     """Test XLA stats are logged using a logger."""
 
     model = BoringModel()
-    xla_stats = XLAStatsMonitor()
-    logger = CSVLogger(tmpdir)
+    with pytest.deprecated_call(match="The `XLAStatsMonitor` callback was deprecated in v1.5"):
+        xla_stats = XLAStatsMonitor()
+
+    class DebugLogger(CSVLogger):
```

Comment:
Reviewer: This follows the same test logic as in `tests/callbacks/test_device_stats_monitor.py::test_device_stats_monitor_tpu`